golang/cosmos/app/app.go

```diff
@@ -837,6 +838,7 @@ type cosmosInitAction struct {
 // Name returns the name of the App
 func (app *GaiaApp) Name() string { return app.BaseApp.Name() }
 
+// CheckControllerInited exits if the controller initialization state does not match `expected`.
```

Comment:
The `godoc` tool doesn't parse markdown.

```suggestion
// CheckControllerInited exits if the controller initialization state does not match expected.
```

Reply:
Is that a problem? In general I prefer delimiting code references with backticks even if they don't get parsed as markdown.